### PR TITLE
chore: add issue and pr templates

### PR DESCRIPTION
**Pull request by toolifelesstocode**
## Description

This adds PR and Issue templates to the repo. They are very similar to https://github.com/eludris/eludris 's templates.

### CODE REVIEW

File: .github/ISSUE_TEMPLATES/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+          required: true
+        - label: Have you checked other issues to make sure this is not a duplicate?
+          required: true
+
+  - type: textarea
+    attributes:
+      label: Minimum Reproducible Steps
+      description: | 
+        Insert any steps to help reproduce this issue
+        Include console output, if any
```
Comment (EnokiUN) [inline .github/ISSUE_TEMPLATES/bug_report.yml:25]:
```suggestion
        Additionally, include console output, if any
```

File: .github/ISSUE_TEMPLATES/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      label: Checks
+      options:
+        - label: Have you checked the documentation?
+          required: true
+        - label: Have you checked other issues to make sure this is not a duplicate?
+          required: true
+
+  - type: textarea
+    attributes:
+      label: Minimum Reproducible Steps
```
Comment (EnokiUN) [inline .github/ISSUE_TEMPLATES/bug_report.yml:22]:
```suggestion
      label: Minimum Reproduction Steps
```

File: .github/ISSUE_TEMPLATES/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      description: | 
+        Insert any steps to help reproduce this issue
+        Include console output, if any
+
+  - type: textarea
+    attributes:
+      label: Screenshots
+      description: | 
+        Insert any images/videos of the issue.
```
Comment (EnokiUN) [inline .github/ISSUE_TEMPLATES/bug_report.yml:31]:
```suggestion
        Insert any images/videos of the issue
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+<!--
+## This is a **Logic Change**
+- [ ] Docs have been updated to reflect these changes if necessary.
+- [ ] Changes have been tested.
+-->
+
+<!--
+## This is a **UI Change**
+- [ ] This has been previewed and's output is as intended
```
Comment (EnokiUN) [inline .github/PULL_REQUEST_TEMPLATE.md:19]:
```suggestion
- [ ] This has been previewed and looks as intended
```

**Review by EnokiUN — CHANGES_REQUESTED** [review]

**Review by EnokiUN — APPROVED** [review]

File: .github/ISSUE_TEMPLATES/bug_report.yml [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+      label: What is the issue?
+      description: Insert a detailed description of what your issue is
+    validations:
+      required: true
+
+  - type: checkboxes
+    attributes:
+      label: Checks
+      options:
+        - label: Have you checked the documentation?
```
Comment (toolifelesstocode) [inline .github/ISSUE_TEMPLATES/bug_report.yml:15]: Pengin has no docs :neckbeard:

**Pull request merged**